### ISSUE TITLE
patternlab-node 2.7.1 breaks CLI

### ISSUE THREAD

**Issue by raphaelokon** [pattern-lab/patternlab-node-cli#27]
### General information

- NodeJS version: 7.4.0

### Reproduction steps

Running the CLI results in 
```
patternlab init
Loading engines from the core...


Loading engines from the edition or test directory...

  mustache: good to go

Done loading engines.


====[ Pattern Lab / Node - v2.7.1 ]====
```

**Comment by bmuenzenmeyer** [comment]
I'll look at this when I can.  Something with incremental builds I guess.

**Comment by bmuenzenmeyer** [comment]
@raphaelokon try again - https://github.com/pattern-lab/patternlab-node/releases/tag/v2.7.2

confirmed working against edition-node-gulp without `npm link`

**Comment by raphaelokon** [comment]
Thanks @bmuenzenmeyer. Fixed by df3602a
